ci/builders/web_engine.json

```diff
@@ -21,7 +30,24 @@
         "tests": []
     },
     {
-        "archives": [],
+        "archives": [
+		{
+			"name": "linux_host_debug_unopt",
+			"include_paths": ["out/host_debug_unopt"],
+			"exclude_paths": [
+				"out/host_debug_unopt/obj",
+				"out/host_debug_unopt/exe.unstripped"
+			]
+		},
+		{
+			"name": "javascript_tests",
```

Comment:
Let's call this `web_tests`. "javascript" is an implementation detail that's only part of the story. We also have WebAssembly, and sources are Dart and C++, so referring to JavaScript would be confusing.